benchmarks/sift/metrics.py

```diff
@@ -35,40 +36,16 @@ def recall(actual_sorted: np.ndarray, results: np.ndarray):
     results: ndarray
         The ANN results
     """
-    len = results.shape[1]    
```

Comment:
this is just ruff?

Reply:
yes, but i moved this to `bench_utils.py` as well, i need it in other places